modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,36 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can connect apps to Anypoint Platform without granting them unrestricted access to data or having to share sensitive credentials.
```

Comment:
OAuth is less about "connecting" apps, more about securely delegating user access to a client application with a restricted set of permissions called scopes. Can we make sure that's conveyed in this paragraph?